lib/parsers/webhook_json_parser.rb

```diff
@@ -99,6 +100,14 @@ def deleted?
         end
       end
 
+      def module_name
```

Comment:
Could module_name reuse the repo_name code (we can expand it for the other vcs's)?

Reply:
no, because repo_name will include the full repo name (I.E. puppet-nginx) where as module_name parses out only the module name

Reply:
We probably should break it out more, but I want to wait till post 1.0.0

Let's focus on getting everything working as expected and cleaned up